NEP-P2P_Protocol_change.mediawiki

```diff
@@ -0,0 +1,129 @@
+<pre>
+  NEP: <to be assigned>
+  Title: P2P Protocol change
+  Author: Shargon and Belane
+  Type: Standard
+  Status: Draft
+  Created: 2018-03-07
+</pre>
+
+==Abstract==
+
+We believe that a P2P protocol optimization is necessary in order to avoid unnecessary data exchange between nodes.
```

Comment:
```suggestion
A P2P protocol optimization is deemed necessary in order to avoid unnecessary data exchange between nodes. 
```
I think a more generic (vs personal "we believe") writing style suits NEPs better as they are quite serious documents (imo).

Reply:
TCP has his own CRC protocol